splice_machine/src/main/java/com/splicemachine/derby/utils/SpliceDateFormatter.java

```diff
@@ -14,29 +14,88 @@
 
 package com.splicemachine.derby.utils;
 
+import com.splicemachine.db.iapi.reference.SQLState;
+import java.sql.SQLException;
+import static java.lang.String.format;
+
 public class SpliceDateFormatter {
 
     private String format = null;
-    private java.time.format.DateTimeFormatter formatter;
+    private java.time.format.DateTimeFormatter formatter = null;
```

Comment:
Could `DateTimeFormatter` be imported to avoid using fully-qualified name here and down the class?

Reply:
yes, thanks.